Makefile

```diff
@@ -0,0 +1,16 @@
+OUT := build
+MPYCROSS := /usr/bin/mpy-cross
+MPYC_FLAGS := -v -v -mcache-lookup-bc
+PYS := $(shell find ./ev3dev2 -type f \( -iname "*.py" ! -iname "setup.py" \))
+MPYS := $(PYS:./%.py=${OUT}/%.mpy)
+vpath %.py .
+
+${OUT}/%.mpy: %.py
+	@mkdir -p $(dir $@)
+	${MPYCROSS} ${MPYC_FLAGS} -o $@ $<
+
```

Comment:
Once this Makefile runs, it overwrites the system-installed library files, right? So undoing it would be force-reinstalling the library packages?

Reply:
Correct. I added a comment at the top of the makefile, as well as more text in CONTRIBUTING.rst to clarify its usage